### DISCUSSION TITLE
LORA modulation options

### DISCUSSION THREAD

**Discussion by KedrMiraj** [sandeepmistry/arduino-LoRa#520] [General]
Hi!
Question about LORA ra-02 ra-01 modules (SX1278)
The instructions for the modules say that they use FSK, GFSK, MSK, GMSK, LoRa and OOK modulation.
How can I switch the modulation modes in the arduino code?
Or is this not possible in your library?
Or by default, modules use LORA(tm) modulation?
If it is possible to switch these modes in other libraries, is there any information in which it is possible?
Thanks for the answer!

**Comment by morganrallen** [comment]
This library only supports the LoRa modulation mode.

**Reply by KedrMiraj** [reply]
Okay, thanks. I couldn't figure out what mode it was working in. Now it became clear

There is only one question left - maybe there is a way to switch the modulation in other libraries?
